README.md

```diff
@@ -48,6 +48,7 @@ If `PCAP_LIBDIR` environment variable is set when building the crate, it will be
 #### `capture-stream`
 
 Use the `capture-stream` feature to enable support for streamed packet captures.
+It's tested on ubuntu and macosx only.
```

Comment:
I think this should say. This feature is *supported* only on ubuntu and macosx. Otherwise, this gives the impression it should work on windows, but just isn't tested.

Reply:
Did it as is. Thanks!